pkg/compute/logstream/live_streamer.go

```diff
@@ -0,0 +1,78 @@
+package logstream
+
+import (
+	"context"
+	"io"
+
+	"github.com/bacalhau-project/bacalhau/pkg/lib/concurrency"
+	"github.com/bacalhau-project/bacalhau/pkg/logger"
+	"github.com/bacalhau-project/bacalhau/pkg/models"
+)
+
+type LiveStreamerParams struct {
+	Reader io.Reader
+	Buffer int
```

Comment:
This is set but I don't think it's used?

Reply:
good catch. fixed in  https://github.com/bacalhau-project/bacalhau/pull/3354/commits/81fe8c5645703d466ae06b2ed828aac4e5fd2bf4